Regression/Checksum/checksum.py

```diff
@@ -243,9 +243,8 @@ def evaluate(self, rtol=1.0e-9, atol=1.0e-40):
             )
             print("Benchmark: %s" % ref_benchmark.data.keys())
             print("Test file: %s" % self.data.keys())
-            print("\n----------------\nNew file for " + self.test_name + ":")
+            print(f"\nNew checksums file {self.test_name}.json:")
             print(json.dumps(self.data, indent=2))
-            print("----------------")
             sys.exit(1)
 
```

Comment:
Please double check if this script by @NeilZaim still works when this output here is changed:
https://warpx.readthedocs.io/en/latest/developers/checksum.html#reset-a-benchmark-from-the-azure-pipeline-output-on-github

Reply:
Thanks for pointing this out. I'm working on a fix in https://github.com/ECP-WarpX/WarpX/pull/5372.